### ISSUE TITLE
Issue: tasks not completing due to async completion

### ISSUE THREAD

**Issue by matt-lio** [Startr/RIAC-Guidebooks#8]
When executing ```bash <(curl -sL startr.sh)``` to run program, this issue arises.
<img width="707" alt="Screenshot 2024-07-25 at 11 17 29 AM" src="https://github.com/user-attachments/assets/00d7b899-3ba5-4b09-b670-039b8b2237fe">

**Comment by opencoca** [comment]
Yes fixing #9 brings us to the intermittent async completion issue. 
 
 Fix is likely https://gulpjs.com/docs/en/getting-started/async-completion/
